File: backend-gin/internal/bin/test_pgsql.go
package bin

import (
	"backend/internal/services/instances"
	"context"
	"fmt"
)

func TestPgsql() {
	// Get redis client
	ctx := context.Background()
	pgClient := instances.GetPgClient()
	defer pgClient.Close()

	// Extract list of tables
	tables, err := pgClient.ListTables(ctx)
	if err != nil {
		fmt.Println(err)
	} else {
		fmt.Println(tables)
	}
}

File: backend-axum/src/bin/test_pgsql.rs
use backend::services::instances::get_pgclient;

#[tokio::main]
async fn main() {
    let pg_clinet = get_pgclient().await;
    let tables = pg_clinet
        .list_tables()
        .await
        .expect("could not extract list of tables");
    println!("{:?}", tables);
}
